src/app/pie-loan/pie-loan.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="chart-container">
+  <canvas  id="myChart" >{{ chart }}</canvas>
```

Comment:
```suggestion
  <canvas  id="myChart">{{ chart }}</canvas>
```
Weird spacing. Was prettier run on this?